tests/robottelo/test_cli.py

```diff
@@ -1,8 +1,13 @@
-import mock
+import six
 import unittest2
 
 from robottelo.cli.base import Base
 
+if six.PY2:
+    import mock
+else:
+    from unittest import mock
```

Comment:
`mock` will be installed as a dependency no matter which Python version is used. Would it be simpler to just `import mock`?